ledgerwallet/manifest_json.py

```diff
@@ -0,0 +1,78 @@
+import json
+import os
+from typing import Dict, Optional
+
+from ledgerwallet import params
+from ledgerwallet.manifest import AppManifest, icon_from_file
+
+
+class AppManifestJson(AppManifest):
+    def __init__(self, filename):
+        with open(filename) as f:
+            self.path = os.path.dirname(filename)
+            self.dic = json.load(f)
+            assert "targetId" in self.dic and "binary" in self.dic
+
+    @property
+    def app_name(self) -> str:
+        return self.dic["name"]
```

Comment:
Should this field be required, and thus be checked in the `__init__`?
Or is it optional, and thus should be checked here, like in the other method?